src/app/components/LiveLabel/index.styles.ts

```diff
@@ -1,13 +1,73 @@
-import { css, Theme } from '@emotion/react';
+import { css, Theme, keyframes } from '@emotion/react';
+
+const pulseAnimation = keyframes({
+  '0%': {
+    opacity: 1,
+  },
+  '50%': {
+    opacity: 0,
+  },
+  '100%': {
+    opacity: 1,
+  },
+});
 
 const styles = {
-  liveLabel: ({ palette, spacings, fontVariants }: Theme) =>
+  liveLabelText: ({ palette, spacings, fontVariants }: Theme) =>
     css({
       color: palette.LIVE_DARK,
       display: 'inline-block',
       marginInlineEnd: `${spacings.FULL}rem`,
       ...fontVariants.sansBold,
     }),
+  liveLabelCircle: ({ palette, spacings }: Theme) =>
+    css({
+      color: palette.LIVE_DARK,
+      borderRadius: '50%',
+      display: 'inline-block',
+      width: '16px',
+      height: '16px',
+      background: `radial-gradient(circle,transparent 53%,hsla(180, 100%, 20%,.5) 53.5%, ${palette.LIVE_DARK} 54%)`,
```

Comment:
Do we need the `hsla` here (that formula - `hsla(180, 100%, 20%,.5)` also generates the LIVE_DARK color, so I would rather only have 1 way of doing that...